tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -146,6 +147,7 @@ export class WebGPUBackend extends KernelBackend {
     this.currentCommandEncoder = null;
     this.currentComputePass = null;
     this.supportTimeQuery = device.features.has('timestamp-query');
+    this.adapterInfo = adapterInfo;
```

Comment:
WebGPUBackend is already very crowded. Can we put this and related methods in a standalone class, e.g., adapter?